src/http/middleware.ts

```diff
@@ -89,3 +89,34 @@ export abstract class Middleware {
    */
   abstract run(): any | void;
 }
+
+export type middlewareToRun = {
+  before_request?: any[];
+  after_request?: any[];
+};
```

Comment:
Is it possible to do:
```
before_request?: Function[];
after_request?: Function[];
```
Just to be more explicit on the types?

Reply:
Done. Even typed the functions for better typing support.